core/Dockerfile

```diff
@@ -1,3 +1,18 @@
+### Build train-sim package (not used in core for now, but eventually will)
+# Will be used for testing
+FROM rust:latest AS train_sim_build_env
+COPY --from=train_sim_dir . .
+
+RUN rustup component add llvm-tools && \
+    rustup component add rustfmt && \
+    rustup component add clippy && \
+    cargo install --locked grcov
+ENV RUSTFLAGS="-Cinstrument-coverage -C link-arg=-fuse-ld=mold"
```

Comment:
`-C link-arg=-fuse-ld=mold`

while I'm all for using a fast linker, you probably need to install `mold`...

`-Cinstrument-coverage`

Is this intended?

Reply:
I have to admit I blindly copied the rust flags from the editoast config. 

Isn't `-Cinstrument-coverage` needed for `rgcov` down the line ?

Reply:
indeed, instrument-coverage only makes sense for coverage measurements